modules/hcp-support-matrix.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/hcp-prepare/hcp-requirements.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-support-matrix"_{context}"]
+= Support matrix for {hcp}
+
+Because {mce} includes the HyperShift Operator, releases of {hcp} align with releases of {mce-short}. For more information, see link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles].
+
+[id="hcp-matrix-mgmt_{context}"]
+== Management cluster support
+
+Any supported standalone {product-title} cluster can be a management cluster. The following table maps {mce-short} versions to the management cluster versions that support them:
+
+.Supported {mce-short} versions for {product-title} management clusters
+[cols="2",options="header"]
+|===
+|Management cluster version |Supported {mce-short} version
+
+|4.14 - 4.15
+|2.4
+
+|4.14 - 4.16
+|2.5
+
+|4.14 - 4.17
+|2.6
+
+|4.15 - 4.17
+|2.7
+|===
+
+[id="hcp-matrix-hc_{context}"]
+== Hosted cluster support
+
+For hosted clusters, no direct relationship exists the management cluster version and the hosted cluster version. The hosted cluster version depends on the HyperShift Operator that is included with your {mce-short} version. The following table maps {mce-short} versions to which hosted cluster versions you can create by using the HyperShift Operator that is associated with that version of {mce-short}:
+
+.Hosted cluster versions that can be created by {mce-short} versions
+[cols="5",options="header"]
+|===
+|Hosted cluster version  |{mce-short} 2.4 |{mce-short} 2.5 |{mce-short} 2.6 |{mce-short} 2.7
+
+|4.14
+|Yes
+|Yes
+|Yes
+|Yes
+
+|4.15
+|No
+|Yes
+|Yes
+|Yes
+
+|4.16
+|No
+|No
+|Yes
+|Yes
+
+|4.17
+|No
+|No
+|No
+|Yes
+|===
+
+[id="hcp-matrix-platform_{context}"]
+== Hosted cluster platform support
+
+The following table indicates which {product-title} versions are supported for each platform of {hcp}. In the table, _Management cluster version_ refers to the {product-title} version where the {mce-short} is enabled:
+
+.Required {product-title} versions for platforms
+[cols="3",options="header"]
+|===
+|Hosted cluster platform |Management cluster version |Hosted cluster version
+
+|{aws-full}
+|4.16 - 4.17
+|4.16 - 4.17
+
+|{ibm-power-title}
+|4.17
+|4.17
+
+|{ibm-z-title}
+|4.17
+|4.17
+
+|{VirtProductName}
+|4.14 - 4.17
+|4.14 - 4.17
+
+|Bare metal
+|4.14 - 4.17
+|4.14 - 4.17
+
+|Non-bare-metal agent machines (Technology Preview)
+|4.16 - 4.17
+|4.16 - 4.17
+|===
+
+[id="hcp-matrix-updates_{context}"]
+== Updates of {mce-short}
+
+When you update to another version of the {mce-short}, your hosted cluster can continue to run if the HyperShift Operator that is included in the version of {mce-short} supports the hosted cluster version. The following table shows the affects of {mce-short} updates on various versions of hosted clusters:
```

Comment:
```suggestion
When you update to another version of the {mce-short}, your hosted cluster can continue to run if the HyperShift Operator that is included in the version of {mce-short} supports the hosted cluster version. The following table shows the effects of {mce-short} updates on various versions of hosted clusters:
```
Same fix for the other uses of "affect" instead of "effect" in this file

Reply:
This table is a bit unclear to me. I'm not sure what to make of the empty rows, should those have some other text in them like "hosted cluster can't run" or "doesn't apply" or something?

Reply:
Does not apply since for example 2.4 cant create future versions of hosted clusters so no impact of upgrade of versions that do not exist.

Reply:
Thanks Laura, looks good :)